### PR TITLE
Added EditIcon to ItemCardCompact | Added some item information as DialogContent in remove dialog

### PR DESCRIPTION
**Pull request by devYaoYH**
- Added `<EditIcon />` so that it is more obvious in Cart Compact view mode that the user can click on an item row to edit the item
- Reduced bottom padding for header
- Added Item Media, Name, Unit as content for removal dialog
- Changed button style to neutral color (text-only buttons)

**Demo**
ScanAndGo GPay (11)

**Edit Icon in Compact View**
Screenshot 2020-07-28 at 14 25 19

### CODE REVIEW

**Comment by BhavyaLight** [comment]
I don't think we need the image in the dialogue box, just title of item should be fine?

**Comment by BhavyaLight** [comment]
The edit icon also looks slightly bigger in size than the text

**Comment by BhavyaLight** [comment]
Check with Dwasan, and the image in dialog is fine - however remove the shadow

**Comment by dhritiwasan14** [comment]
Hey! Great work with the implementation. Can we remove the shadow and make sure the right text is made bold in the card? Also, I updated the mocks with the strings that we would like to use in this case. Drop in your feedback!

File: client/src/pages/ScanStore/ScanStore.tsx [hunk 1]
```diff
@@ -166,12 +170,16 @@ function ScanStore() {
...
     setShowRemoveConfirmation(false);
   };
 
   const raiseRemoveConfirmation = (barcode: string) => {
-    setRemoveBarcode(barcode);
+    // Find the item information for what is to be deleted
+    const itemToRemove = cartItems
+      .map((cartItem) => cartItem.item)
+      .find((item) => item.barcode === barcode);
+    if (!itemToRemove) return; // Should not be the case here
```
Comment (cszqwe) [inline client/src/pages/ScanStore/ScanStore.tsx:181]: If this would never happen, can just ignore

Comment (devYaoYH) [reply client/src/pages/ScanStore/ScanStore.tsx:181]: during runtime it shouldn't be the case, but static compile-time check raises error for the next line, hence this check. Updated comment to explain the case here

**Comment by devYaoYH** [comment]
@dhritiwasan14 Typography fixes are underway in #234 which addresses various font-related issues. (and will resolve the bold-face hierarchy). I've Updated the title text to reflect the mocks and used black for text-color.
Could I confirm that the 'REMOVE' text should be bolded?
[WIP] currently looking into overriding defaults to make dialog box look more rounded

File: client/src/pages/ScanStore/ScanStore.tsx (hunk 1 above)
Comment (cszqwe) [reply client/src/pages/ScanStore/ScanStore.tsx]: I see.

**Pull request merged**